server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/ProjectDashboardPage.java

```diff
@@ -17,80 +17,76 @@
-import static com.codeborne.selenide.Selenide.$;
-import static com.codeborne.selenide.Selenide.$$;
 import static org.assertj.core.api.Assertions.assertThat;
 
+
 public class ProjectDashboardPage {
 
   public ProjectDashboardPage() {
-    $(".overview").shouldBe(visible);
+    Selenide.$(".overview").shouldBe(Condition.visible);
   }
 
   public SelenideElement getLinesOfCode() {
-    SelenideElement element = $("#overview-ncloc");
-    element.shouldBe(visible);
+    SelenideElement element = Selenide.$("#overview-ncloc");
+    element.shouldBe(Condition.visible);
     return element;
   }
 
   public SelenideElement getLanguageDistribution() {
-    SelenideElement element = $("#overview-language-distribution");
-    element.shouldBe(visible);
+    SelenideElement element = Selenide.$("#overview-language-distribution");
+    element.shouldBe(Condition.visible);
     return element;
   }
 
   public SelenideElement getOverviewMeasure(String measure) {
-    ElementsCollection measures = $$(".overview-domain-measure");
-    SelenideElement element = measures.find(text(measure)).shouldBe(visible);
-    return element;
+    ElementsCollection measures = Selenide.$$(".overview-domain-measure");
+    return measures.find(Condition.text(measure)).shouldBe(Condition.visible);
   }
 
-  private SelenideElement getTagsMeta() {
-    SelenideElement element = $(".overview-meta-tags");
-    element.shouldBe(visible);
+  private static SelenideElement getTagsMeta() {
+    SelenideElement element = Selenide.$(".overview-meta-tags");
+    element.shouldBe(Condition.visible);
     return element;
   }
 
   public ProjectDashboardPage shouldHaveTags(String... tags) {
     String tagsList = String.join(", ", Arrays.asList(tags));
-    this.getTagsMeta().$(".tags-list > span").should(hasText(tagsList));
+    this.getTagsMeta().$(".tags-list > span").should(Condition.text(tagsList));
     return this;
   }
 
   public ProjectDashboardPage shouldNotBeEditable() {
     SelenideElement tagsElem = this.getTagsMeta();
-    tagsElem.$("button").shouldNot(exist);
-    tagsElem.$("div.multi-select").shouldNot(exist);
+    tagsElem.$("button").shouldNot(Condition.exist);
+    tagsElem.$("div.multi-select").shouldNot(Condition.exist);
     return this;
   }
 
   public ProjectDashboardPage shouldBeEditable() {
     SelenideElement tagsElem = this.getTagsMeta();
-    tagsElem.$("button").shouldBe(visible);
+    tagsElem.$("button").shouldBe(Condition.visible);
     return this;
   }
 
   public ProjectDashboardPage openTagEditor() {
     SelenideElement tagsElem = this.getTagsMeta();
-    tagsElem.$("button").shouldBe(visible).click();
-    tagsElem.$("div.multi-select").shouldBe(visible);
+    tagsElem.$("button").shouldBe(Condition.visible).click();
+    tagsElem.$("div.multi-select").shouldBe(Condition.visible);
     return this;
   }
 
   public SelenideElement getTagAtIdx(Integer idx) {
     SelenideElement tagsElem = this.getTagsMeta();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2209)